services/src/test/java/org/keycloak/theme/ResourceLoaderTest.java

```diff
@@ -21,16 +21,16 @@ public class ResourceLoaderTest {
     public void testResource() throws IOException {
         String parent = "dummy-resources/parent";
         assertResourceAsStream(parent, "myresource.css", true, true);
-        assertResourceAsStream(parent, NONE + "myresource.css", false, true);
-        assertResourceAsStream(parent, SINGLE + "myresource.css", false, false);
-        assertResourceAsStream(parent, DOUBLE + "myresource.css", false, false);
+        assertResourceAsStream(parent, NONE + "forbidden.css", false, true);
+        assertResourceAsStream(parent, SINGLE + "forbidden.css", false, false);
+        assertResourceAsStream(parent, DOUBLE + "forbidden.css", false, false);
 
         assertResourceAsStream(parent, "one/" + NONE + "myresource.css", true, true);
-        assertResourceAsStream(parent, "one/" + SINGLE + "myresource.css", false, false);
-        assertResourceAsStream(parent, "one/" + DOUBLE + "myresource.css", false, false);
+        assertResourceAsStream(parent, "one/" + SINGLE + "forbidden.css", false, false);
+        assertResourceAsStream(parent, "one/" + DOUBLE + "forbidden.css", false, false);
```

Comment:
These should be something like

```suggestion
        assertResourceAsStream(parent, "one/" + SINGLE + SINGLE + "forbidden.css", false, false);
        assertResourceAsStream(parent, "one/" + DOUBLE + DOUBLE + "forbidden.css", false, false);
```

otherwise they are looking in the wrong spot for forbidden.css

You should also remove dummy-resources/myresource.css to clarify the test more.

Reply:
Results from the other branch that has win UT enabled:
https://github.com/Pepo48/keycloak/actions/runs/18324870437/job/52187218549